stress_test_model_loan_book.R

```diff
@@ -121,7 +124,7 @@ scenarios <- c(
   "B2DS",
   "CPS",
   "NPS",
-  "NPSRTS",
+  # "NPSRTS",
```

Comment:
I would prefer removing this instead of commenting it out

Reply:
I concur